### PR TITLE
Find wireless interface for FreeBSD

### PR DESCRIPTION
**Pull request by pkgdemon**

### CODE REVIEW

**Comment by dwfreed** [comment]
I don't see a need to change this file, as it's not used in any way as far as I know.  These scripts just serve as example initscripts.

**Comment by williamh** [comment]
@pkgdemon If this is a better way of detecting a wireless interface for FreeBSD, I'm willing to consider merging it because people might use it as an example.

**Comment by williamh** [comment]
Please fix the script to use $() instead of backticks. This is the more common way  of handling command substitution.

**Comment by pkgdemon** [comment]
The sysctl -b net.wlan.devices is the only way I know of to detect wireless on FreeBSD.  The /dev/net/* option does not work on FreeBSD as net does not exist under /dev/.  I will look at removing the ` backticks to verify I can still make it work.  Thanks.

**Comment by pkgdemon** [comment]
The changes have been made to remove the backticks.

File: init.d.misc/wpa_supplicant.in [hunk 1]
```diff
@@ -39,6 +39,11 @@ find_wireless()
 			fi
 		done
 		;;
+	FreeBSD)
+		for iface in $(sysctl -b net.wlan.devices) 2>/dev/null`; do
```
Comment (dwfreed) [inline init.d.misc/wpa_supplicant.in]: Errant backtick here, where the right parenthesis should be.  (ie, the line should look like this:

```sh
for iface in $(sysctl -b net.wlan.devices 2>/dev/null); do
```

**Comment by pkgdemon** [comment]
@dwfreed I fixed, and also moved the other ) after null to match the other cases.

**Pull request closed without merging**